app/services/irs_api_service.rb

```diff
@@ -11,7 +11,7 @@ def self.df_return_sample
   end
 
   def self.import_federal_data(authorization_code, state_code)
-    unless Rails.env.production? || Rails.env.staging?
+    unless Rails.env.production?
```

Comment:
Again - should it be `acts_like_production?`

Reply:
also same as above